runtime/service_test.go

```diff
@@ -323,3 +327,42 @@ func TestDebugConfig(t *testing.T) {
 		assert.Equal(t, c.service.config.DebugHelper.GetFirecrackerLogLevel(), cfg.LogLevel)
 	}
 }
+
+// TestTaskCreateDoesNotLogStdoutOrStderrFields validates the firecracker-containerd runtime service implementation
+// does not log the Stdout/Stderr request fields because they may contain sensitive information for the shim logger binary use case.
+func TestTaskCreateDoesNotLogStdoutOrStderrFields(t *testing.T) {
+	testLogger, hook := test.NewNullLogger()
+	testLogger.Level = logrus.DebugLevel
+
+	vmIsReady := make(chan struct{})
+	close(vmIsReady)
```

Comment:
`defer close(vmisReady)`?

Reply:
In this case, the channel is closed after the `CreateVM` API call is made. So mocking here the VM is ready.